### DISCUSSION TITLE
PLAYR - Building an iOS iptv player / list manager

### DISCUSSION THREAD

**Discussion by jamestapping** [iptv-org/community#949] [Show and tell]
I've been building a pretty simple(looking) app for IPTV fans, kind of works on an iPad, but is really made for iPhone.
It's in beta at the moment via TestFlight. Here's the link.

It's basic but working, so please be kind, but any ideas, thoughts welcome.

https://testflight.apple.com/join/GMnd6gA6

edit: now available on the app store look for "playr - iptv player"

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/awesome-iptv/pull/282
